src/utils/utils.js

```diff
@@ -0,0 +1,82 @@
+//select all from a table
+const selectAll = (aTable) => `SELECT * FROM ${aTable}`;
```

Comment:
The variable names in this file make the functions quite difficult to read. For example, this variable could just be `table`.

Reply:
thank you for the feedback I have amended the code